authority/admin/api/handler.go

```diff
@@ -38,7 +40,7 @@ var mustAuthority = func(ctx context.Context) adminAuthority {
 }
 
 // Route traffic and implement the Router interface.
-func Route(r api.Router, acmeResponder ACMEAdminResponder, policyResponder PolicyAdminResponder) {
+func Route(r api.Router, acmeResponder ACMEAdminResponder, policyResponder PolicyAdminResponder, webhookResponder WebhookAdminResponder) {
```

Comment:
This is ok, but a `ops... RouteOption` alternative similar to the above might be nice.

Reply:
d48f6740